projects/openexr/openexr_exrenvmap_fuzzer.cc

```diff
@@ -56,8 +58,9 @@ static char *buf_to_file(const char *buf, size_t size) {
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
-  char *file = buf_to_file((const char *)data, size);
-  if (!file) return 0;
+  char *file = fuzzer_get_tmpfile(data, size);
```

Comment:
Since this is a C++ file, you should use the `FuzzerTemporaryFile` class instead, and then you don't have to worry about releasing it at the end.